cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,43 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('Article', () => {
-  before(() => {
+import ArticlePageObject from '../support/pages/article.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
 
-  });
+const articlePage = new ArticlePageObject();
+const signInPage = new SignInPageObject();
 
-  beforeEach(() => {
-    cy.task('db:clear');
+describe('Article', () => {
+  before(() => {
+    cy.visit('/');
+    signInPage.visit();
+    signInPage.typeEmail('riot@qa.team');
+    signInPage.typePassword('12345Qwert!');
+    signInPage.clickSignInBtn();
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(2000);
```

Comment:
Using `cy.wait(2000)` is generally discouraged as it can lead to flaky tests. It's better to wait for specific elements or conditions using commands like `cy.get()` or `cy.contains()` with `.should('be.visible')` to ensure the application is in the expected state before proceeding.